examples/paho-mqtt/README.md

```diff
@@ -22,13 +22,14 @@ For example this one for debian base linux users
 
 - To connect to a broker, use the `con` command:
 ```
-con <ipv6 addr> <port> <keepalivetime> <client ID> <user> <password>
+con <ipv6 addr> [user] [password] [port] [keepalivetime]
```

Comment:
What about `Client ID`, it is described but not in the synopsis.

Reply:
I'm user the same as UserID

Reply:
I can add it but don't see it relevant. where you able to use it with IPv4?